src/routes/popup/transaction/[id].tsx

```diff
@@ -356,7 +372,50 @@ export default function Transaction({ id: rawId, gw, message }: Props) {
                     {browser.i18n.getMessage("transaction_from")}
                   </PropertyName>
                   <PropertyValue>
-                    {formatAddress(transaction.owner.address, 6)}
+                    <div>
```

Comment:
Eventually, it would be better to turn this into a bunch of individual components.

In any case, the reason for this change is that before the UI would only prompt users to add the "to" address as contact. After this change, it would prompt for both "to" and "from", and it would also take into account if an address is yours and display "(you)" next to it.